irf/collection_area.py

```diff
@@ -1,15 +1,97 @@
 import numpy as np
+
 from astropy.stats import binom_conf_interval
 import astropy.units as u
+from astropy.table import Table
+
+from scipy.ndimage.filters import gaussian_filter
+
+from irf.oga import calculate_fov_offset
```

Comment:
what is oga?

Reply:
Short for open gamma-ray astro format. I'll add a module docstring.